boa/cli/validate.py

```diff
@@ -0,0 +1,33 @@
+from jsonschema import validate
+from jsoncomment import JsonComment
+import os
+
+from rich.console import Console
+
+console = Console()
+
+
+def schema_dir():
+    return os.path.join(os.path.dirname(__file__), "../../schemas")
```

Comment:
Any reason to go with os.path instead of making this py3k only and usong pathlib?
I guess it would take care of my second comment of avoiding *nix specific slashes.

Reply:
right. haven't yet gotten used to pathlib but should educate myself asap :)

Reply:
if you want to, that would be exciting :) I am sure there is plenty of os.path.join to fix in boa, hehe.